src/gui/mzroll/mzfileio.cpp

```diff
@@ -583,16 +574,8 @@ void mzFileIO::fileImport(void) {
             QString filename = samples.at(i);
             mzSample* sample = loadSample(filename);
             if (sample) {
-                    sample->enumerateSRMScans();
-                    sample->calculateMzRtRange();    //set min and max values for rt
-                    sample->fileName = filename.toStdString();
-
-                    if ( filename.contains("blank",Qt::CaseInsensitive))
```

Comment:
@rish9511 This should be removed from the else block as well, then

Reply:
@shubhra-agrawal 
Thanks for pointing out